Feisty/GameView/GameViewComponents/GamesViewController.swift

```diff
@@ -46,8 +44,27 @@ class GamesViewController: UITableViewController {
     }
 
     let gameDetails = dataViewModel.getGameDetails(at: indexPath.row)
-    cell.txtViewGameName?.text = gameDetails.name
+    cell.txtViewGameName?.text = gameDetails.gameName
     cell.txtGamePrice?.text = gameDetails.gamePrice
+    
+    cell.cellTappedAction = { [weak self] in
+      
+      let storyboard = UIStoryboard(name: "Main", bundle: .main)
+      let detailView = storyboard.instantiateViewController(withIdentifier: "GameDetailsViewController") as? GameDetailsViewController
```

Comment:
Here's a helpful NSObject extension you can use to avoid writing strings here:

```
public extension NSObject {
    var className: String {
        return String(describing: type(of: self)).components(separatedBy: ".").last!
    }

    class var className: String {
        return String(describing: self).components(separatedBy: ".").last!
    }
}
```

That way, you can say self.className instead of having to constantly type the String and update it if you change class name